webpack/components/extensions/HostDetails/DetailsTabCards/HwPropertiesCard.js

```diff
@@ -71,7 +72,9 @@ const HwPropertiesCard = ({ isExpandedGlobal, hostDetails }) => {
         </DescriptionListGroup>
         <DescriptionListGroup>
           <DescriptionListTerm>{__('RAM')}</DescriptionListTerm>
-          <DescriptionListDescription>{memory}</DescriptionListDescription>
+          <DescriptionListDescription>{NumberToHumanSize(memory*1024, {
+          strip_insignificant_zeros: true, precision: 2
+        })}</DescriptionListDescription>
         </DescriptionListGroup>
         <DescriptionListGroup>
           <HostDisks totalDisks={totalDisks} />
```

Comment:
I just noticed this on the screenshot. This value is the total number of bytes of all disks combined, not the number of disks.

Reply:
I have that fixed in a separate pr